### PR TITLE
chanfunding: remove unsupported linter directive.

### PR DESCRIPTION
**Pull request by ziggie1984**
Remove unsupported directive: `level=warning msg="[runner/nolint] Found unknown linters in //nolint directives: ifelsechain"`

### CODE REVIEW

File: lnwallet/chanfunding/coin_select.go [hunk 1]
```diff
@@ -373,7 +373,7 @@ func CoinSelectUpToAmount(feeRate chainfee.SatPerKWeight, minAmount, maxAmount,
 	)
 
 	var errInsufficientFunds *ErrInsufficientFunds
-	if err == nil { //nolint:gocritic,ifElseChain
+	if err == nil { //nolint:gocritic
```
Comment (guggero) [inline lnwallet/chanfunding/coin_select.go:376]: We could also just turn this into a `switch` statement which might allow us to remove this remaining nolint directive too?

Comment (ziggie1984) [reply lnwallet/chanfunding/coin_select.go:376]: great point

**Review by guggero — APPROVED** [review]
Nice, LGTM :tada:

**Review by yyforyongyu — APPROVED** [review]
LGTM🎉

**Pull request merged**